sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1935,7 +1935,7 @@ object GpuOverrides {
       ExprChecks.fullAgg(
         TypeSig.LONG, TypeSig.LONG,
         repeatingParamCheck = Some(RepeatingParamCheck(
-          "input", TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL, TypeSig.all))),
+          "input", pluginSupportedOrderableSig, TypeSig.all))),
```

Comment:
This confuses me. Even if the signatures are the same, there is no reason why count needs input to be orderable. Also why limit count to single level structs, why not all structs or lists/etc? Am I missing something?

Reply:
> This confuses me. Even if the signatures are the same, there is no reason why count needs input to be orderable. 

If it's about the naming, I anticipated this question but forgot to rename the variable. 

> Also why limit count to single level structs, why not all structs or lists/etc? 

I think it's the same line of thinking as why only the single-level  structs were enabled for sort. This would be the first step to establish single-level as a baseline and then have a follow to allow deeper or arbitrary nesting. I believe it was suggested to me previously by @sameerz . Maybe it's no longer relevant.